Code/Tools/AssetProcessor/Platform/Windows/assetprocessor_windows_files.cmake

```diff
@@ -7,6 +7,8 @@
 #
 
 set(FILES
+    native/FileWatcher/FileWatcher_platform.h
     native/FileWatcher/FileWatcher_win.cpp
+    native/FileWatcher/FileWatcher_windows.h
```

Comment:
We should keep the file names names consistent between header and source file.

PAL dictates the file should be `FileWatcher_windows.cpp`, so I think we should change the .cpp file

Reply:
Fixed.